lib/schema-registry.js

```diff
@@ -424,16 +415,10 @@ SchemaRegistry.prototype._registerSchemaLatest = Promise.method(function (schema
 
   log.debug('_registerSchemaLatest() :: Registered schema:', schemaObj.topic);
 
```

Comment:
Removing `this.valueSchemas` and `this.keySchemas` in favor of single schema container `this.schemas`